templates/web/common/includes/off_days_2024_banner.tt.html

```diff
@@ -0,0 +1,33 @@
+<!-- start templates/[% component.name %] -->
+
+[% banner_off_days_2024_top = BLOCK %]
+
+<section id="off-days-2024-banner-top" class="off-days-banner">
+</section>
+
+<script>
+    const offDaysBannerID = document.getElementById('off-days-2024-banner-top');
+
+    function getBannerCookie(bcname) {
+        const name = bcname + '=';
+        const decodedCookies = decodeURIComponent(document.cookie);
+        const cookies = decodedCookies.split(';');
+        for (const cookie of cookies) {
+        let c = cookie;
+        while (c.charAt(0) == ' ') { c = c.substring(1); }
+        if (c.indexOf(name) == 0) { return c.substring(name.length, c.length); }
+        }
+        
+        return '';
+    }
+
+    if (getBannerCookie('off_donation_banner_2023_a') !== '') {
+        console.log('cookie found');
+        offDaysBannerID.style.display = 'flex';
+    } else {
+        console.log('cookie not found');
+        offDaysBannerID.style.display = 'none';
+    }
+</script>
```

Comment:
This code works, but i need to reload page to display off days banner.
Do you have an idea to reload only banners ? 
@alexgarel

Reply:
As discussed, this is not a big deal, banner can be displayed on next page, the invitation is really aimed at frequent contributors.